docs/release-notes/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Release Notes
+---
+
+## Latest:
+* **Saros for Eclipse (Saros/E)**: [15.0.0](saros-e_15.0.0.md)
```

Comment:
This link does not work in my local setup. Change to `html` instead of `md`? Same goes for link to Saros/I release notes.
```suggestion
* **Saros for Eclipse (Saros/E)**: [15.0.0](saros-e_15.0.0.html)
```